test/fixtures/api-accepts.js

```diff
@@ -341,6 +340,8 @@ function stubFiles(sandbox) {
   files.getComponentPath.withArgs('missing').returns('missingThing');
   files.getComponentPath.withArgs('invalid').returns(null);
 
+  sandbox.stub(files, 'getComponents').returns(['clay-c5', 'clay-c3', 'clay-c4']);
```

Comment:
i have no idea what this does...but this file removes logging?

Reply:
No, it just makes the function `getComponents` return this array when called from API tests.

Reply:
Oh, it removes the stub for the old logger from the test file, not from anything else. API tests are looking for the response objects, not what was logged. Individual files do that.